### PR TITLE
Add context for the NedelecSZ element

### PR DESCRIPTION
**Pull request by dangars**
@tamiko I would like to do maxwell simulations with deal.ii. It would be good to explain a little bit more what is wrong with the former Nedelec element and in which cases the former Nedelec element leads to wrong results.

Is it possible to use adaptive mesh refinement with the new NedelecSZ element?

### CODE REVIEW

File: 9.1/paper.tex [hunk 1]
```diff
@@ -242,14 +242,34 @@ \section{Overview}
   N{\'e}d{\'e}lec element on quadrilaterals and hexahedra. It is based on
   the work of Zaglmayr \cite{Zag06} and overcomes the sign conflict issues
   present in traditional N{\'e}d{\'e}lec elements that arise from the edge
-  and face parameterizations used in the basis functions. Therefore, this
+  and face parameterizations used in the basis functions.
+  Because of the sign conflict it is only possible to use the traditional
+  N{\'e}d{\'e}lec element either in a grid configuration or in a complex
+  tetrahedra mesh configuration. It is not possible to use traditional
+  N{\'e}d{\'e}lec in a complex hexahedra mesh configuration. This
```
Comment (bangerth) [inline 9.1/paper.tex]: I don't think that's generically true. It's true that the *implementation in deal.II* has difficulties with this, though that is not the fault of "the Nedelec element".

File: 9.1/paper.tex [hunk 2]
```diff
@@ -242,14 +242,34 @@ \section{Overview}
...
   element should provide consistent results for general quadrilateral and
```
Comment (bangerth) [inline 9.1/paper.tex]: Maybe qualify what "this element" refers to?

File: 9.1/paper.tex [hunk 3]
```diff
@@ -242,14 +242,34 @@ \section{Overview}
...
-  hexahedral elements for which the relative orientations of edges and
+  hexahedral elements in a complex mesh configuration
+  for which the relative orientations of edges and
   faces (as seen from all adjacent cells) are often difficult to establish.
   The \texttt{FE\_NedelecSZ} element addresses the sign conflict problem by
   assigning a globally defined orientation to local edges and faces. A
   detailed overview of the implementation of the \texttt{FE\_NedelecSZ}
   element in \dealii{} can be found in \cite{Kynch2017}.
+  
+  \textbf{
```
Comment (bangerth) [inline 9.1/paper.tex]: Put these things in a question on the mailing list, or in a github issue. This isn't the right place for a discussion :-)

**Comment by dangars** [comment]
@bangerth You are right. This shouldn't be a PR :)

I close this PR, but I leave my branch active in case @tamiko would like the text I wrote taking into account @bangerth's comments

**Comment by bangerth** [comment]
On 6/9/19 12:09 PM, Daniel Garcia-Sanchez wrote:
> I close this PR, but I leave my branch active in case @tamiko 
> <https://github.com/tamiko> would like the text I wrote taking into 
> account @bangerth <https://github.com/bangerth>'s comments

Don't wait for anyone else to pick text out of closed pull requests :-) 
If you want something changed, propose the text in a PR!

**Comment by dangars** [comment]
> On 6/9/19 12:09 PM, Daniel Garcia-Sanchez wrote: I close this PR, but I leave my branch active in case @tamiko <https://github.com/tamiko> would like the text I wrote taking into account @bangerth <https://github.com/bangerth>'s comments
> Don't wait for anyone else to pick text out of closed pull requests :-) If you want something changed, propose the text in a PR!

That is right :) I could reopen this PR. But the text I proposed is just a minor suggestion.

I see that the paper is ready for submission!

I think that it is better to keep the PR closed.

**Pull request closed without merging**